### DISCUSSION TITLE
Server chunks/blocks are not loading completely

### DISCUSSION THREAD

**Discussion by bencobi** [itzg/docker-minecraft-server#1892] [Q&A]
Hi,
I installed the minecraft server on my Synology NAS DS923+ in docker. Minecraft Container with 6GB RAM.
If I run about 2 minutes in one direction, the world stops loading the blocks. It's not only on my client. My friends got the same issue on my server. Does anyone have any idea what this could be?

**Comment by DevDroid1** [comment]
i have the same prob

**Reply by DevDroid1** [reply]
@itzg

**Reply by itzg** [reply]
Why the ping? This is not a docker specific problem.
